data/systemd/anaconda-nm-disable-autocons-rhel.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+ConditionKernelCommandLine=|ip
+ConditionKernelCommandLine=|inst.ks
+ConditionOSRelease=ID=rhel
+Description=NetworkManager autoconnections configuration for Anaconda installation environment for RHEL
+Before=NetworkManager.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/anaconda-nm-disable-autocons
+
```

Comment:
Extra new line not needed. Feel free to ignore.

Reply:
Fixed.